modules/tinymce/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The content of the `contenteditable="false"` element could be selected with the mouse on Firefox  #TINY-8828
 - Parsing large documents no longer throws a `Maximum call stack size exceeded` exception #TINY-6945
 - DomParser filter matching was not checked between filters, which could lead to an exception in the parser #TINY-8888
+- Lists with `contenteditable="false"` can no longer be toggled, and `contenteditable="true"` list elements within them can no longer be indented, split into another list element, or appended to the previous list element by deletion. #TINY-8920
```

Comment:
Should this include the `.` other items doesn't seem to have that except the Deprecated one. Also the log item is rather long not sure if we care anymore. I think the details should go into release notes instead. This would more be an overview that lists now works better with contenteditable=false.

Reply:
Fixed, thank you

Reply:
@MitchCTiny what you had was correct actually with the fullstop. Remember @brian-forte had asked us to include it and if you look at all the released versions it does have fullstops. I thought Millie was going to pass that on to the Husky team but I guess that got missed before she left.

Either way, not something we need to enforce but it saves us having to fix it later.

Reply:
Ah right, I remember. Fixed

Reply:
Time to make that changelog linter and add a punctuation rule. :smile: